src/walletSdk/Anchor/Sep24.ts

```diff
@@ -212,38 +214,20 @@ export class Sep24 {
     const toml = await this.anchor.sep1();
     const transferServerEndpoint = toml.transferServerSep24;
 
-    let qs: { [name: string]: string } = {};
```

Comment:
basically just moving this code to a helper function so can be used for both seps